src/app/websites/website-services/afterdark/components/afterdark-submission-form/afterdark-submission-form.component.html

```diff
@@ -0,0 +1,13 @@
+<form [formGroup]="formGroup">
+  <div formGroupName="options" class="row no-gutters section">
+    <div class="row no-gutters">
+      <div class="col-md col-lg col-xl">
+        <div class="row no-gutters">
+          <div class="col-md col-lg col-xl">
+            <folder-select formControlName="folders" [folders]="folders" [multiple]="true"></folder-select>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</form>
```

Comment:
Where is the Tags input?
See inkbunny-submission-form.component.html and inkbunny-submission-form.component.ts

Reply:
Added a tags input to this component.